lib/ansible/modules/system/systemd.py

```diff
@@ -321,6 +321,11 @@ def main():
         ),
         supports_check_mode=True,
         required_one_of=[['state', 'enabled', 'masked', 'daemon_reload']],
+        required_by=dict(
+            state=('name'),
```

Comment:
Same here: if you want a `tuple`, you need to add a fake comma (or however that thing is called :D ).

(There are some more, also in other files.)

Reply:
Support by this code: https://github.com/ansible/ansible/pull/28662/files#diff-90085fdcec6ed8b273ba885eaee60328R1678